rollup.config.js

```diff
@@ -1,35 +1,41 @@
-import {createRequire} from 'node:module';
+import typescript from '@rollup/plugin-typescript';
+import { createRequire } from 'node:module';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+// import terser from '@rollup/plugin-terser';
+import dts from 'rollup-plugin-dts';
+import terser from '@rollup/plugin-terser'
+
 const require = createRequire(import.meta.url);
 const pkg = require('./package.json');
-import terser from '@rollup/plugin-terser';
-import babel from '@rollup/plugin-babel';
-import dts from 'rollup-plugin-dts';
 
+const commonPlugins = [
+  typescript({ tsconfig: './tsconfig.json' }),
+  commonjs(),
+    // terser()
```

Comment:
Hey @murbanowicz thanks for the PR - looking awesome 🚀 

Just wondering if there was a reason for removing minification?

Reply:
Hi Mate! Look at the bottom part of the PR description. I would ask you to help on this and manage rollup. I've spent quite a lot of time on this already which makes it pointless on my side.

Reply:
@murbanowicz my bad, missed that part of the description. Looking into it now :)

We've updated the types in the PKCE vanilla JS lib this depends on now also which may make some of the typings here better

Reply:
It seems that this issue is with upgrading terser from `0.1.0` to `0.2.0` as per [this issue](https://github.com/rollup/plugins/issues/1366) 

If you revert the change in `package.json` back to `"@rollup/plugin-terser": "^0.1.0"` then it all builds fine :) 

Hopefully they resolve soon so we can use the latest, but I'm ok with it being on 0.1.0 for now